packages/typespec-azure-rulesets/src/rulesets/data-plane.ts

```diff
@@ -41,6 +41,8 @@ export default {
     "@azure-tools/typespec-azure-core/no-private-usage": true,
     "@azure-tools/typespec-azure-core/friendly-name": true,
     "@azure-tools/typespec-azure-core/no-query-explode": true,
+    "@azure-tools/typespec-client-generator-core/require-client-suffix": true,
+    "@azure-tools/typespec-client-generator-core/property-name-conflict": true,
```

Comment:
yeah also this is actually included in this PR, so either disable for now or need to fix the spec repos.

Also can you move into its own section(just comment above)